### ISSUE TITLE
"Pick color" tool doesn't update the color preview in the color panel

### ISSUE THREAD

**Issue by QuentinBouvier** [viliusle/miniPaint#139]
Hello.

The **pick color** tool will successfully pick a color and apply it to the next tool use, but will not fully update the **colors** panel preview. The fields are updated, the color is selected, the rectangle preview remains in its previous state.

The screenshot shows a green rectangle for a red color selected.

![Green color for red color selected, miniPaint](https://user-images.githubusercontent.com/32564163/56192233-3c6c3f80-602e-11e9-91ef-a54befbac1d3.png)

Thanks for your work on MiniPaint. I just started fooling around with it and it looks very great.

**Comment by viliusle** [comment]
Thank you for testing, but i can not reproduce it, what is your browser, browser version, OS? Also can you check console for errors?

**Comment by QuentinBouvier** [comment]
Hi Vilius.

I tested it on Firefox dev (67.0b9 (64-bit)) and Chrome (73.0.3683.103 (official build) (64 bits)). There's no error in the console on both browsers. It was reported by a coworker on Chrome and Firefox too (I don't know which version he uses, but most probably recent). All that on windows 10 with node 10.1 and npm 6.9.

I tried it on local, freshly cloned repo and on [your  github.io site](https://viliusle.github.io/miniPaint).

Here's a gif, because I just discovered how to do make it and I consider only communicating with animated capture of my screen from now. 

![malfunction_miniPaint](https://user-images.githubusercontent.com/32564163/56344453-c26ac080-61bd-11e9-85b8-ecb1cd7141e6.gif)
